pkgs/development/libraries/gamin/default.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation (rec {
   # <sys/socket.h> with Glibc 2.9.
   configureFlags = "--disable-debug --with-python=${python} CPPFLAGS=-D_GNU_SOURCE";
 
-  patches = [ ./deadlock.patch ] ++ map fetchurl (import ./debian-patches.nix);
+  patches = [ ./deadlock.patch ./returnval.patch ] ++ map fetchurl (import ./debian-patches.nix);
```

Comment:
It looks like this additional patch is triggering a mass-rebuild, but it doesn't look darwin-specific. So the PR should probably be re-opened against staging.